perses/rjmc/geometry.py

```diff
@@ -455,13 +455,14 @@ def _logp_propose(self, top_proposal, old_positions, beta, new_positions=None, d
         if self.use_sterics:
             platform_name = 'CPU' # faster when sterics are in use
```

Comment:
we don't use this in the _canonical_ system setup regime, but perhaps platform_name if self.use_sterics should also be set to 'CUDA'